### ISSUE TITLE
Zeus frequencies

### ISSUE THREAD

**Issue by NemesisRE** [michail-nikolaev/task-force-arma-3-radio#982]
We have set our frequencies by defining the tf_freq_\* variables

```
tf_freq_west = [0,7,["310","320","330","340","350","360","370","380","30.6"],0,"WEST",8,0,nil,false];
tf_freq_west_lr = [0,7,["84.4","31","32","33","34","35","36","37","38","30,6"],0,"WEST",9,0,false];
tf_freq_east = [0,7,["410","420","430","440","450","460","470","480","40.6"],0,"EAST",8,0,nil,false];
tf_freq_east_lr = [0,7,["74.4","41","42","43","44","45","46","47","48,""40.6"],0,"EAST",9,0,false];
tf_freq_guer = [0,7,["510","520","530","540","550","560","570","580","50.6"],0,"GUER",8,0,nil,false];
tf_freq_guer_lr = [0,7,["64.4","51","52","53","54","55","56","57","58","50.6"],0,"GUER",9,0,false];
```

and want that they are set for our forced curator too, is this possible?
If it is possible then how?

regards
NemesisRE

**Comment by NemesisRE** [comment]
another problem is when we have defined the encryptionkey in this array and the curator unit doesn't use this settings, zeus is not able to communitcate with anyone via radio.

**Comment by michail-nikolaev** [comment]
Hm... Looks like Zeus not supports such logic currently....

**Comment by michail-nikolaev** [comment]
I'll check later in more details - please ping me if no result for 1-2 days

**Comment by NemesisRE** [comment]
friendly ping

**Comment by michail-nikolaev** [comment]
Oh, looks like not easy thing.... Currenty Zeus just uses frequencies settings from radios... So, you could try yo do a trick. 
Use API to set freqs and code for next radios:

[TF_curator_backpack_1, "TF_curatorBackPack"]
[TF_curator_backpack_2, "TF_curatorBackPack"]
[TF_curator_backpack_3, "TF_curatorBackPack"]

**Comment by NemesisRE** [comment]
I'm sure you will find a way in the future :) 
Could you make a more extensive example I'm not sure if I understood the workaround correctly.

Thanks for your great work on this Mod!

regards
NemesisRE

**Comment by NemesisRE** [comment]
@dedmen I saw that you pushed the development a lot, maybe you could have another look at this issue

Our Problem is that frequencies set for the factions are not set for the curator(s). Also if we don't use our frequencyset and instead using the tf_same_sw_frequencies_for_side setting the curator has it's very own.

Currently I solved it by this bullshit https://github.com/quies-in-proelium/qipTPL/blob/master/Functions/zeus/fn_tfrZeus.sqf

**Comment by dedmen** [comment]
That really shouldn't happen :x 
#Devnotes 
could be that allGroups doesn't include virtual curators.
fnc_processGroupFrequencySettings add allCurators to allGroups.

**Comment by dedmen** [comment]
Confirmed
https://community.bistudio.com/wiki/allGroups "Does not contain groups of sideLogic" which are curator's.
Second issue... When TFAR_fnc_processGroupFrequencySettings is first called BIS_fnc_listCuratorPlayers doesn't return the curators.. for some whatever reason.. And at second call of processGroupFrequencySettings the Curators Radios already Initialized with his frequencies

**Comment by dedmen** [comment]
It's a bit ugly but it works. Fixed in 1.0. Remember Changed API in 1.0 for default Frequencies.
Your 
```
tf_freq_west = [0,7,["310","320","330","340","350","360","370","380","30.6"],0,"WEST",8,0,nil,false];
tf_freq_west_lr = [0,7,["84.4","31","32","33","34","35","36","37","38","30,6"],0,"WEST",9,0,false];
tf_freq_east = [0,7,["410","420","430","440","450","460","470","480","40.6"],0,"EAST",8,0,nil,false];
tf_freq_east_lr = [0,7,["74.4","41","42","43","44","45","46","47","48,""40.6"],0,"EAST",9,0,false];
tf_freq_guer = [0,7,["510","520","530","540","550","560","570","580","50.6"],0,"GUER",8,0,nil,false];
tf_freq_guer_lr = [0,7,["64.4","51","52","53","54","55","56","57","58","50.6"],0,"GUER",9,0,false];
```
turns into
```
TFAR_defaultFrequencies_sr_west = ["310","320","330","340","350","360","370","380","30.6"];
TFAR_defaultFrequencies_lr_west = ["84.4","31","32","33","34","35","36","37","38","30,6"];
TFAR_defaultFrequencies_sr_east = ["410","420","430","440","450","460","470","480","40.6"];
TFAR_defaultFrequencies_lr_east = ["74.4","41","42","43","44","45","46","47","48,""40.6"];
TFAR_defaultFrequencies_sr_independent = ["510","520","530","540","550","560","570","580","50.6"];
TFAR_defaultFrequencies_lr_independent = ["64.4","51","52","53","54","55","56","57","58","50.6"];
```
If you want to use these changes earlier feel free to help test us Pre-releases. 1.0-dev.139 should have the fix. For better contact to Devs see #1116
